packages/grid/src/GridMetricCalculator.ts

```diff
@@ -1691,11 +1712,11 @@ class GridMetricCalculator {
   /**
    * Sets the width for the specified column
    * @param column The column model index to set
-   * @param size The size to set it to, or null to reset the column size
+   * @param size The size to set it to, or undefined to reset the column size
    */
   setColumnWidth(column: ModelIndex, size?: number): void {
     const userColumnWidths = new Map(this.userColumnWidths);
-    if (size != null) {
+    if (size !== undefined) {
```

Comment:
Did you check usages from JS code to make sure none explicitly pass `null`? Could leave this as `!= null` and TS shouldn't complain I think. That way any JS stuff that hasn't been converted yet still works. Those files would still complain if you were calling w/ explicit null once it's converted to TS.

Reply:
Yea I'm just going to force calling into `resetRowHeight` instead, and require a number here.

All places that call `setRowHeight`/`setColumnWidth` pass a value